docs/_assets/style.css

```diff
@@ -1,23 +1,53 @@
 body[layout='layout-home'] .markdown-body .call-to-action:nth-of-type(2) {
   --primary-color: #222;
```

Comment:
As i said in your previous MR: "Overriding variables here will break the usage of theming (dark mode)." 
Don't use variables, but set the values directly.